openshift/official_template.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: log-anomaly-detector
+  namespace: aiops-prod-log-analysis
+objects:
+- apiVersion: v1
+  kind: PersistentVolumeClaim
+  metadata:
+    labels:
+      app: log-anomaly-detector-f20
+    name: log-anomaly-detector-f20
+  spec:
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+    storageClassName: ${STORAGE_CLASS}
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    labels:
+      app: log-anomaly-detector
+    name: log-anomaly-detector
+  spec:
+    lookupPolicy:
+      local: true
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      app: log-anomaly-detector
+    name: anomaly-detector
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: log-anomaly-detector:latest
+    source:
+      git:
+        ref: master
+        uri: https://github.com/AICoE/log-anomaly-detector.git
+      type: Git
+    strategy:
+      sourceStrategy:
+        from:
+          kind: ImageStreamTag
+          name: python:3.6
+          namespace: openshift
+      type: Source
+    triggers:
+    - imageChange: {}
+      type: ImageChange
+    - type: ConfigChange
+- apiVersion: apps.openshift.io/v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      app: log-anomaly-detector-f20
```

Comment:
Why does deployment config name mention `f20`? What does it mean?

Reply:
So for each team that onboards this service and share the same namespace we should have unique names. In this case we use f20 its just a placeholder.

Reply:
Let me be more clear here - f20 should be a parameter (as it was in original template which you are removing in this PR) exaclty for a reason you mentioned - we need to have unique names for each deployed LAD

Reply:
Fixed it. Done.